### PR TITLE
Updating formats to be consistent at a code level

### PR DESCRIPTION
**Pull request by Chrislee187**

### CODE REVIEW

**Comment by Seraphiana** [comment]
all looks fine to me 👍

File: app/models/vehicle_recalls_and_faults_alert.rb [hunk 1]
```diff
@@ -5,16 +5,16 @@ class VehicleRecallsAndFaultsAlert < Document
...
-    :fault_type,
-    :faulty_item_type,
-    :alert_issue_date,
-    :manufacturer,
-    :faulty_item_model,
-    :serial_number,
-    :build_start_date,
-    :build_end_date,
-  ]
+  FORMAT_SPECIFIC_FIELDS = %i(
```
Comment (Rosa-Fox) [inline app/models/vehicle_recalls_and_faults_alert.rb:8]: 👍

File: app/views/attachments/_attachment_links.html.erb [hunk 2]
```diff
@@ -12,7 +12,8 @@
         <% @document.attachments.each do |attachment| %>
             <li class="attachment">
               <span class="title"><%= attachment.title %></span>
-              <%= link_to "edit", edit_document_attachment_path(params[:document_type_slug], @document.content_id, attachment.content_id) %>
+              <%= link_to "edit", edit_document_attachment_path(params[:document_type_slug], @document.content_id,
```
Comment (Rosa-Fox) [inline app/views/attachments/_attachment_links.html.erb]: I would prob do:

```
<li class="attachment">
              <span class="title"><%= attachment.title %></span>
              <%= link_to "edit",
                edit_document_attachment_path(params[:document_type_slug],
                @document.content_id,
                attachment.content_id)
              %>
              <%= attachment.snippet %>
            </li>
```

Comment (Rosa-Fox) [inline app/views/attachments/_attachment_links.html.erb]: I would probably do like [this](https://gist.github.com/Rosa-Fox/37a0bf217c23736cec9c3ad84b9eda37) as separates it out a bit more.

File: app/views/attachments/edit.html.erb [hunk 3]
```diff
@@ -4,7 +4,8 @@
 
 <%= render(partial: "shared/attachments_form", locals: {
     attachment: @attachment,
-    url: document_attachment_path(params[:document_type_slug], params[:document_content_id], params[:attachment_content_id]),
+    url: document_attachment_path(params[:document_type_slug], params[:document_content_id],
```
Comment (Rosa-Fox) [inline app/views/attachments/edit.html.erb]: Maybe split up tiny bit more: https://gist.github.com/Rosa-Fox/96b1b21b484617cb5153eb12b9772a65

File: app/views/documents/_actions.html.erb [hunk 4]
```diff
@@ -14,7 +14,8 @@
         <%= presenter.publish_text %>
 
         <% if presenter.publish_button_visible? %>
-          <button name="submit" class="btn btn-danger" data-module="confirm" data-message="<%= presenter.publish_alert %>" data-disable-with="Publishing..." >Publish</button>
+          <button name="submit" class="btn btn-danger" data-module="confirm" data-message="<%= presenter.publish_alert %>"
```
Comment (Rosa-Fox) [inline app/views/documents/_actions.html.erb]: Not sure how much difference it makes but this way seems slightly more readable: https://gist.github.com/Rosa-Fox/d1482f3137599a1c8e4603da88d719a0

File: app/views/documents/edit.html.erb [hunk 5]
```diff
@@ -1,14 +1,16 @@
 <% content_for :breadcrumbs do %>
     <li><%= link_to "Your documents", documents_path(current_format.slug) %></li>
-    <li><%= link_to @document.title, document_path(document_type_slug: params[:document_type_slug], content_id: @document.content_id) %></li>
+    <li><%= link_to @document.title, document_path(document_type_slug: params[:document_type_slug],
```
Comment (Rosa-Fox) [inline app/views/documents/edit.html.erb:3]: 👍

File: app/views/documents/show.html.erb [hunk 6]
```diff
@@ -64,7 +64,8 @@
           <% if i == truncate_at_count - 1 %>
             <tr class="table-header">
               <td colspan="3" class="js-toggle-target text-center">
-                <strong><a href="#expand-table" class="js-toggle">…and <%= attachments.count - (truncate_at_count - 1) %> more</a></strong>
+                <strong><a href="#expand-table" class="js-toggle">…and
```
Comment (Rosa-Fox) [inline app/views/documents/show.html.erb:67]: https://gist.github.com/Rosa-Fox/20869ee6675aaf67dfd948560dbf360f

File: app/views/manual_sections/_form.html.erb [hunk 7]
```diff
@@ -4,7 +4,8 @@
     <%= f.label :title %>
     <%= f.text_field :title, class: 'form-control' %>
     <%= f.label :summary %>
-    <%= f.text_area :summary, class: 'form-control short-textarea js-length-counter', data: { :"count-message-threshold" => 280, :"count-message-selector" => ".summary-length-info" } %>
+    <%= f.text_area :summary, class: 'form-control short-textarea js-length-counter',
```
Comment (Rosa-Fox) [inline app/views/manual_sections/_form.html.erb:7]: Probably no point in these changes to manuals code as it is being removed soon.

**Comment by Chrislee187** [comment]
Changes made as per your notes Rosa, are there any actual guidelines on coding style type stuff anywhere?

**Comment by Rosa-Fox** [comment]
Hi,

There are the [Style Guides](https://github.com/alphagov/styleguides) though some aspects, such as this are still open to interpretation. I remember when I created the Trello card a while back that I was told to break the longer lines on to separate lines to make it as readable at first glance as possible, which is why I thought it would be good to break them up a little more.

**Pull request merged**